src/App.tsx

```diff
@@ -36,6 +37,7 @@ export default function App() {
               <Route path='/login' element={<Login />}/>
               <Route path='/register' element={<Register />}/>
               <Route path='/verify/:adminId' element={<Verify />}/>
+              <Route path='/verify/:adminId/verifyCollab' element={<VerifyCollab />} />
```

Comment:
Maybe we can make the `<VerifyCollab />` component into a more general password reset component? Basically the exact same code just renaming the component something like `<ResetPassword />` and changing the path to something like `'/reset/:adminId'`